site/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
❓ Why do we have to add this? (Or is this a file move? hard to tell)

Reply:
Vite requires the HTML file to be in the root.